.github/workflows/perftest.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - name: Deploy workload
         run: |
-          kubectl create deploy k6 --image kennethreitz/httpbin --port 80 && \
+          kubectl create deploy k6 --image ghcr.io/sharat87/httpbun --port 80 && \
```

Comment:
so if we build this image, we can probably in a followup replace it here.

As a suggestion, raise a different PR just to change ci.yaml and the images/httpbun files, we get this image created and promoted and start using those on our tests. It will take a bit more time to get this one merged, but then we will be inside the whole k8s infra stuff.